layers/peterborough.map

```diff
@@ -2,6 +2,7 @@ MAP
   NAME "Peterborough"
   STATUS ON
   EXTENT -180 -90 180 90
+  SHAPEPATH "../../layers"
```

Comment:
If this is added, I guess it should (needs to? not sure) be removed from the existing highways layer.

Reply:
I wasn't sure if `SHAPEPATH` was specific to `DATA`, but I've checked and `CONNECTION` works either way, so I've changed it to the short version in 8af282484f69c4f1ebd6874a4874a639516fd469.